projects/graph/src/graph.py

```diff
@@ -61,6 +61,30 @@ def dft_recursive(self, starting_v_id, visited=None):
         for neighbor in self.vertices[starting_v_id]:
             self.dft_recursive(neighbor, visited)
 
+    def bfs(self, starting_v_id, destination_v_id):
```

Comment:
Nice implementation of BFS! The differences between DFS and BFS are not drastic so it would be great if there were more comments to describe the sequence of events in this early stage of using BFS/DFS. Overall nice job though!